pyexchange/uniswapv2_analytics.py

```diff
@@ -327,13 +320,22 @@ def get_trades(self, pair: str, page_number: int = 1) -> List[Trade]:
         mints_in_last_two_days = list(filter(lambda mint: int(mint['timestamp']) > two_days_ago_unix, mint_events))
 
         if current_liquidity != Wad.from_number(0) and len(mints_in_last_two_days) >= 1:
-            start_timestamp = mints_in_last_two_days[-1]['timestamp']
+            if self.our_last_pair_hour_timestamp > mints_in_last_two_days[-1]['timestamp']:
+                start_timestamp = self.our_last_pair_hour_timestamp
```

Comment:
For these three blocks, would it be more readable to use `max` function instead of a conditional?  For example, this could replace four lines of code:
```
start_timestamp = max(self.our_last_pair_hour_timestamp, mints_in_last_two_days[-1]['timestamp'])
```

Reply:
Yeah, that would be much cleaner. I will update accordingly.